src/configure_settings/configure_settings.jl

```diff
@@ -9,6 +9,7 @@ function default_settings()
         "CapacityReserveMargin" => 0,
         "CO2Cap" => 0,
         "StorageLosses" => 1,
+        "VirtualChargeDischargeCost" => 1,
```

Comment:
It would be useful to note here in a comment in that unlike the other the settings which are basically binary values, this thing has units of `$/MWh`.